change/@fluentui-react-positioning-4bd82184-0e7b-46e8-a681-c87900b25568.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 131 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26593&commitId=658cf2763a0883154956f5b53e25a8634751e3fa&env=prod)

✅ There was 0 screenshots added, 39 screenshots removed, 1701 screenshots unchanged, 0 screenshots with different dimensions and 92 screenshots with visible difference.


  <details><summary><b>unknown</b> 131 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Combobox Converged.Appearance- filled-darker.focused.chromium.png|242679| Changed|
|Combobox Converged.Appearance- filled-lighter.focused.chromium.png|242971| Changed|
|Combobox Converged.Appearance- outline (default).focused.chromium.png|243663| Changed|
|Combobox Converged.Appearance- underline.focused.chromium.png|243505| Changed|
|Combobox Converged.Disabled option.default.chromium.png|271979| Changed|
|Combobox Converged.Invalid- filled-darker.focused.chromium.png|242679| Changed|
|Combobox Converged.Invalid- filled-lighter.focused.chromium.png|242971| Changed|
|Combobox Converged.Invalid- outline.focused.chromium.png|243663| Changed|
|Combobox Converged.Invalid- underline.focused.chromium.png|243505| Changed|
|Combobox Converged.Open with grouped options.default.chromium.png|316262| Changed|
|Combobox Converged.Open with inlinePopup.default.chromium.png|271961| Changed|
|Combobox Converged.Open.default.chromium.png|271949| Changed|
|Combobox Converged.Option with long content.default.chromium.png|300082| Changed|
|Combobox Converged.Size- large.focused.chromium.png|244603| Changed|
|Combobox Converged.Size- small.focused.chromium.png|241871| Changed|
|Combobox Converged.With multiselect value.focused.chromium.png|243663| Changed|
|Combobox Converged.With placeholder.focused.chromium.png|243607| Changed|
|Combobox Converged.With value.focused.chromium.png|243531| Changed|
|Dropdown Converged.Disabled option.default.chromium.png|243818| Changed|
|Dropdown Converged.Open with grouped options.default.chromium.png|243818| Changed|
|Dropdown Converged.Open with inlinePopup.default.chromium.png|243831| Changed|
|Dropdown Converged.Open.default.chromium.png|243818| Changed|
|Dropdown Converged.Option with long content.default.chromium.png|243818| Changed|
|InfoButton.default - Dark Mode.active.chromium.png|9101| Changed|
|InfoButton.default - High Contrast.active.chromium.png|10983| Changed|
|InfoButton.default.active.chromium.png|31423| Changed|
|InfoButton.size.rest.chromium.png|87581| Changed|
|Menu Converged - basic.default - RTL.chromium.png|1607616| Changed|
|Menu Converged - basic.default.chromium.png|1607624| Changed|
|Menu Converged - groups.default - Dark Mode.chromium.png|1555773| Changed|
|Menu Converged - groups.default - High Contrast.chromium.png|1561185| Changed|
|Menu Converged - groups.default - RTL.chromium.png|1606761| Changed|
|Menu Converged - groups.default.chromium.png|1606770| Changed|
|Menu Converged - icon slotted content.default - RTL.chromium.png|1607408| Changed|
|Menu Converged - icon slotted content.default.chromium.png|1607422| Changed|
|Menu Converged - nested submenus.default - RTL.chromium.png|1607297| Changed|
|Menu Converged - nested submenus.default.chromium.png|1607317| Changed|
|Menu Converged - secondary content.default - RTL.chromium.png|1607476| Changed|
|Menu Converged - secondary content.default.chromium.png|1607493| Changed|
|Menu Converged - selection groups.default - RTL.chromium.png|1606715| Changed|
|Menu Converged - selection groups.default.chromium.png|1606760| Changed|
|Menu Converged - selection.checkbox - Dark Mode.chromium.png|1572902| Changed|
|Menu Converged - selection.checkbox - High Contrast.chromium.png|1579396| Changed|
|Menu Converged - selection.checkbox - RTL.chromium.png|1607616| Changed|
|Menu Converged - selection.checkbox.chromium.png|1607642| Changed|
|Menu Converged - split item.default - RTL.chromium.png|1607381| Changed|
|Menu Converged - split item.default.chromium.png|1607401| Changed|
|Menu Converged - submenuIndicator slotted content.default - RTL.chromium.png|1607874| Changed|
|Menu Converged - submenuIndicator slotted content.default.chromium.png|1607891| Changed|
|Menu nested within a MenuTrigger.default.chromium.png|1606820| Changed|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|65592| Changed|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|325426| Changed|
|Popover Converged.positioning.positioned popovers.chromium.png|325361| Changed|
|Positioning.arrow - RTL.chromium.png|1600858| Changed|
|Positioning.arrow.chromium.png|1600857| Changed|
|Positioning.auto size.chromium.png|1603723| Changed|
|Positioning.coverTarget - RTL.chromium.png|1603501| Changed|
|Positioning.coverTarget.chromium.png|1603515| Changed|
|Positioning.disable tether.chromium.png|1605494| Changed|
|Positioning.horizontal flip - RTL.chromium.png|1606144| Changed|
|Positioning.horizontal flip.chromium.png|1606144| Changed|
|Positioning.horizontal overflow - RTL.chromium.png|1606143| Changed|
|Positioning.horizontal overflow.chromium.png|1606143| Changed|
|Positioning.offset - RTL.chromium.png|1601543| Changed|
|Positioning.offset function - RTL.chromium.png|1601543| Changed|
|Positioning.offset function.chromium.png|1601543| Changed|
|Positioning.offset.chromium.png|1601543| Changed|
|Positioning.pinned.chromium.png|1606733| Changed|
|Positioning.position and align props - RTL.chromium.png|1601543| Changed|
|Positioning.position and align props.chromium.png|1601543| Changed|
|Positioning.position fixed - RTL.chromium.png|1601543| Changed|
|Positioning.position fixed.chromium.png|1601543| Changed|
|Positioning.reset target.chromium.png|1605082| Changed|
|Positioning.vertical flip.chromium.png|1606425| Changed|
|Positioning.vertical overflow.chromium.png|1606448| Changed|
|Positioning.visibility modifiers.has  [data-popper-escaped]  when the popper escapes the reference element's boundary.chromium.png|1594307| Changed|
|Positioning.visibility modifiers.has  [data-popper-is-intersecting]  when the popover intersects boundaries.chromium.png|1589132| Changed|
|Positioning.visibility modifiers.has  [data-popper-reference-hidden]  when the reference is hidden.chromium.png|1591186| Changed|
|Tooltip Converged.basic - Dark Mode.chromium.png|1582283| Changed|
|Tooltip Converged.basic - High Contrast.chromium.png|1583269| Changed|
|Tooltip Converged.basic.chromium.png|1608142| Changed|
|Tooltip Converged.inverted - Dark Mode.chromium.png|1582496| Changed|
|Tooltip Converged.inverted withArrow - Dark Mode.chromium.png|1582846| Changed|
|Tooltip Converged.inverted withArrow.chromium.png|1604981| Changed|
|Tooltip Converged.inverted.chromium.png|1606655| Changed|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|40124| Changed|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|205177| Changed|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|205169| Changed|
|Tooltip Converged.text-wrapping.chromium.png|1607533| Changed|
|Tooltip Converged.withArrow - Dark Mode.chromium.png|1582336| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|1583260| Changed|
|Tooltip Converged.withArrow.chromium.png|1608014| Changed|
|Combobox Converged.Disabled option.active option.chromium.png|0| Removed|
|Combobox Converged.Disabled option.hover.chromium.png|0| Removed|
|Combobox Converged.Open with grouped options.active option.chromium.png|0| Removed|
|Combobox Converged.Open with grouped options.hover.chromium.png|0| Removed|
|Combobox Converged.Open with inlinePopup.active option.chromium.png|0| Removed|
|Combobox Converged.Open with inlinePopup.hover.chromium.png|0| Removed|
|Combobox Converged.Open.active option.chromium.png|0| Removed|
|Combobox Converged.Open.hover.chromium.png|0| Removed|
|Combobox Converged.Option with long content.active option.chromium.png|0| Removed|
|Combobox Converged.Option with long content.hover.chromium.png|0| Removed|
|Dropdown Converged.Disabled option.active option.chromium.png|0| Removed|
|Dropdown Converged.Disabled option.hover.chromium.png|0| Removed|
|Dropdown Converged.Open with grouped options.active option.chromium.png|0| Removed|
|Dropdown Converged.Open with grouped options.hover.chromium.png|0| Removed|
|Dropdown Converged.Open with inlinePopup.active option.chromium.png|0| Removed|
|Dropdown Converged.Open with inlinePopup.hover.chromium.png|0| Removed|
|Dropdown Converged.Open.active option.chromium.png|0| Removed|
|Dropdown Converged.Open.hover.chromium.png|0| Removed|
|Dropdown Converged.Option with long content.active option.chromium.png|0| Removed|
|Dropdown Converged.Option with long content.hover.chromium.png|0| Removed|
|Menu Converged - basic.default - RTL.hover menuitem.chromium.png|0| Removed|
|Menu Converged - basic.default.hover menuitem.chromium.png|0| Removed|
|Menu Converged - icon slotted content.default - RTL.hover menuitem.chromium.png|0| Removed|
|Menu Converged - icon slotted content.default.hover menuitem.chromium.png|0| Removed|
|Menu Converged - nested submenus.default - RTL.all open.chromium.png|0| Removed|
|Menu Converged - nested submenus.default.all open.chromium.png|0| Removed|
|Menu Converged - secondary content.default - RTL.hover menuitem.chromium.png|0| Removed|
|Menu Converged - secondary content.default.hover menuitem.chromium.png|0| Removed|
|Menu Converged - selection groups.default - RTL.selected.chromium.png|0| Removed|
|Menu Converged - selection groups.default.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox - Dark Mode.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox - High Contrast.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox - RTL.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox.selected.chromium.png|0| Removed|
|Menu Converged - split item.default - RTL.submenu open.chromium.png|0| Removed|
|Menu Converged - split item.default.submenu open.chromium.png|0| Removed|
|Menu Converged - submenuIndicator slotted content.default - RTL.submenus open.chromium.png|0| Removed|
|Menu Converged - submenuIndicator slotted content.default.submenus open.chromium.png|0| Removed|
|Menu nested within a MenuTrigger.default.submenu open.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>